sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -842,7 +944,7 @@ export class DatabaseResponse extends ResourceResponse<DatabaseDefinition & Reso
 
 // @public
 export class Databases {
-    constructor(client: CosmosClient, clientContext: ClientContext);
+    constructor(client: CosmosClient, clientContext: ClientContext, encryptionManager?: EncryptionManager);
```

Comment:
This constructor needs to be marked hidden/internal. Users cannot instantiate this class as clientContext is internal class.

Reply:
done